app/root.tsx

```diff
@@ -30,12 +30,13 @@ export const meta: MetaFunction = ({ error }: any) => {
   return [{ title: title }];
 };
 
-export const links: LinksFunction = () => {
-  return [
-    { rel: 'icon', type: 'image/svg-xml', href: faviconAssetUrl },
-    { rel: 'stylesheet', href: stylesheet },
-  ];
-};
+export const links: LinksFunction = () => [
+  { rel: 'icon', type: 'image/svg-xml', href: faviconAssetUrl },
+  { rel: 'stylesheet', href: stylesheet },
+  { rel: 'preconnect', href: 'https://fonts.googleapis.com'},
+  { rel: 'preconnect', href: 'https://fonts.gstatic.com', crossOrigin: 'anonymous' },
+  { rel: 'stylesheet', href: 'https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap' }, 
+];
```

Comment:
Not a blocker: Should we add all of these font styles? If we need only three of them is it better to download them?🙂